frontend/src/pages/main/ContactUs.js

```diff
@@ -0,0 +1,86 @@
+import "./styles/ContactUs.css";
+import Map from "../../components/Map";
+import { useState } from "react";
+const ContactUs = () => {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('')
+  const [selectedCountry, setSelectedCountry] = useState('');
+  const [userMessage, setUserMessage] = useState('');
+  const NAME_regEx = '^[A-Za-z]+';
+  const MAIL_regEx = '^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$';
+
+  const countries = [
+    {
+      'name': 'Talera, Bundi',
+      'value': 'Talera-Bundi'
+    },
+    {
+      'name': 'Keshoraipatan, Bundi',
+      'value': 'keshoraipatan-bundi'
+    },
+    {
+      'name': 'Bundi',
+      'value': 'bundi'
+    },
+    {
+      'name': 'Ladpura, Kota',
+      'value': 'ladpura-kota'
+    },
+    {
+      'name': 'Kota',
+      'value': 'kota'
+    }
+  ]
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    alert(`Form submitted with input: 
```

Comment:
remove `is` after first name.
Capitalize first letter of all the field names of this alert message.